src/txmempool.h

```diff
@@ -677,7 +677,7 @@ class CTxMemPool
      *  limitDescendantSize = max size of descendants any ancestor can have
      *  errString = populated with error reason if any limits are hit
      *  fSearchForParents = whether to search a tx's vin for in-mempool parents, or
-     *    look up parents from mapLinks. Must be true for entries not in the mempool
+     *    look up parents from CTxMemPoolEntry::m_parents. Must be true for entries not in the mempool
```

Comment:
This is just my opinion, but I think it's more helpful to refer to the specific parameter here. Same thing for the other docs.
```suggestion
     *    look up parents from entry.m_parents. Must be true for entries not in the mempool
```

Reply:
I think the suggestion is very good in general. But let me just show what I would actually love:

```xml
* look up parents from <see cref="entry.m_parents"/>. Must be true for entries not in the mempool
```
(Syntax [borrowed](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#see) from C#)

This is, of course, just made up syntax but it would be great to reference parameter's member (`m_parents`) in Doxygen. The only problem is that I'm unable to find the syntax for Doxygen, if it actually supports it.

Do you possibly know?

Reply:
:tada: `\link CTxMemPoolEntry::m_children updateIt.m_children \endlink` seems to do the job.

Resources:
* https://www.doxygen.nl/manual/autolink.html
* https://www.doxygen.nl/manual/examples/autolink/html/class_autolink___test.html

Reply:
Not sure if there have been discussions about this, but my feeling is that most devs read the source rather than the actual doxygen-generated file (at least I do), so I don't like special commands like  `\link ... \endlink` that improve the doxygen-generated documentation, but come at the cost of disturbing the reading flow when looking at the source.

Reply:
I understand your point of view. An alternative point of view is that doxygen can warn us that a reference no longer exists so that it can be fixed in a PR and it would not be necessary to dig where something changed and why (this PR).

Personally, I don't think it would be that bad to have `\link ... \endlink` (I agree it's not visually nice). However, we already use doxygen syntax so a clear rule explaining why this is "too much" should be presented I think. 

Having said that, I can remove it if you insist but I would rather not.